app/scripts/views/mixins/checkbox-mixin.js

```diff
@@ -7,9 +7,10 @@
  */
 
 define([
+  'fxa-checkbox',
   'jquery',
   'lib/strings'
-], function ($, Strings) {
+], function (FxACheckbox, $, Strings) {
```

Comment:
Not a huge fan of camel-casing the `A` here and it conflicts with other usage:

```
$ git grep FxA -- '*.js' | wc -l
      17
$ git grep Fxa -- '*.js' | wc -l
     300
```

(the existing 17 instances are of it are all in comments and strings)